advocacy_docs/edb-postgres-ai/console/estate/monitor_aws.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Monitoring AWS resources in EDB Postgres AI 
+navTitle: Cloud Hosted Databases - AWS resources
+description: How to monitor AWS resources in EDB Postgres AI Estate.
+deepToC: true
+---
+
+## Overview 
+
+Setting up the EDB Postgres® AI Console to monitor your RDS instances and S3 buckets on AWS involves adding a specific policiy and role in AWS. Once these are configured, you need to enter the role ARN of the newly created role into the **Cloud Hosted Databases** wizard, accessible via the **Estate** page in the EDB Postgres AI Console. 
+
+Using this role ARN and a custom policy, the EDB Postgres AI server will have access to the RDS and S3 information in your AWS account. 
+
+After configuring the wizard with the role ARN, you will see the selected AWS resources (RDS instances and/or S3 buckets) in the chosen AWS regions on your **Estate** page in the **Cloud Hosted Databases** section.
+
+## Setting up monitoring of AWS resources in EDB Postgres AI Estate
+
+### Starting the Cloud Hosted Databases wizard
+
+1. Log into **EDB Postgres AI Console**. 
+
+2. Scroll down to the **Cloud Hosted Databases** section, select the **Manage Access** button, and choose your project.
+
+3. The **Cloud Hosted Databases** wizard shows **Step 1 - Create custom policy**
+
+### Creating the AWS custom policy
+
+4. Log into your AWS account with the RDS instances and S3 buckets you want to monitor.
+
+5. Navigate to IAM, and in the navigation pane on the left side of the AWS console, select **Policies**.
+
+6. On the **Policies** dashboard page, select the **Create policy**. button
+
+7. In the **Policy editor** section, choose the JSON option.
+
+8. Type or paste the following JSON policy document into the JSON editor:
+
+```json
```

Comment:
I think we could indent this whole code snippet under the 8. step by adding four blank spaces at the beginning of each line.

Reply:
agreed